src/styles/base.css

```diff
@@ -916,11 +884,24 @@ a.card:hover {
     border-radius: 4px;
 }
 
+.modalview .content .main table {
+    border: 1px solid silver;  
+}
+  
+.modalview .content .main td {
+    padding: 1em;
+}
+
+#searchpage .Cols3 {
+  column-count: 1;
+}
+
+/* From atlassian batch.css - https://wiki.jenkins.io/s/f68dfafb2b4588f7b31742327b4469ae-CDN/en_GB/6441/82994790ee2f720a5ec8daf4850ac5b7b34d2194/be65c4ed0984ca532b26983f5fc1813e/_/download/contextbatch/css/_super/batch.css?atlassian.aui.raphael.disabled=true */
```

Comment:
do we need it always?

Reply:
I think so, it was previously loading the entire css file every time, which was really messing everything up, so I just pulled in the ~6 lines it needed.